src/System.Private.Uri/tests/FunctionalTests/UriRelativeResolutionTest.cs

```diff
@@ -22,6 +22,9 @@ public class UriRelativeResolutionTest
         private const string FullBaseUriGetLeftPart_Authority = "http://user:psw@host:9090";
         private const string FullBaseUriGetLeftPart_Query = "http://user:psw@host:9090/path1/path2/path3/fileA?query";
 
+        // A few of these tests depend on bugfixes made in .NET Framework 4.7.1 and must be skipped on older versions.
+        public static bool IsNotNetfx47OrOlder => !PlatformDetection.IsFullFramework || PlatformDetection.IsNetfx471OrNewer;
```

Comment:
I used a negative comparison so that I could disable the tests via `ConditionalFact`, which takes a single method that returns a boolean.

The fixes here predate all of my work on System.Uri, so I'm not sure exactly when they made it in. I'll try to find the exact fixes to verify the version.

Reply:
> used a negative comparison so that I could disable the tests via ConditionalFact, which takes a static method that returns a boolean

ConditionalFact/Theory can take a property  that would return TRUE to run the test.  So "not 4.7 or older" == "4.71 or newer".  You should be able to omit using the confusing "not" syntax.

Reply:
Will using IsNetfx471OrNewer exclude .NET Core? I did initially consider it, but thought that it would.

Reply:
That sounds good. I'll implement it that way (and update it to 4.7.2).

Reply:
Testing took longer than expected, but this should be fixed in my latest commit.